README.md

```diff
@@ -123,9 +123,11 @@ import-prices --supplier SERCO/GEOAMEY --year 2020
 import-reports --from YYYY-MM-DD --to YYYY-MM-DD
 ```
 ```
-# Bulk price adjustments
+# Annual price adjustments
 
-bulk-price-adjustment --supplier SERCO/GEOAMEY --effective-year 2021 --multiplier 1.12 --force
+inflationary --supplier SERCO/GEOAMEY --effective-year 2021 --multiplier "1.5" --details "inflation based" --force
```

Comment:
The readme changes reflect the change to the CLI commands.  There are now two distinct types.